### PR TITLE
Update ocean variables tuto

### PR DESCRIPTION
**Pull request by Marie59**
Update of Ocean variables tuto. 1 workflow added with 2 tools following the first ODV tool. Plus, it matches the recent update made on ODV and divand tools.

As usual, no test ;)

1. Give your pull request a **good title**
    - e.g. "Fix typo in ansible-galaxy tutorial" or "Add new transcriptomics tutorial covering a new sequencing technology"
2. Describe your changes in detail, list anything you still need some help with or things that are still TODO
3. Check that your images are allowed to be re-hosted by the GTN!
4. Still working on it? Make it a **Draft** pull request
   - Once it is ready, choose **Ready for Review**
   - Then the automated tests will run and we will know to review and merge it

### CODE REVIEW

**Comment by Marie59** [comment]
I think that except for the missing test all checks passed !:)

File: topics/climate/tutorials/ocean-variables/tutorial.md [hunk 1]
```diff
@@ -96,7 +91,7 @@ Once you finish your work on ODV, if you want to retrieve data and/or the entire
 >    - *"Select if you are using an ODV collection in a zip folder or if you have your own raw data"*: `The data you are using are an ODV collection in a zip folder`
 >    - *"ODV collection in a zip folder."*: `Eutrophication_Med_profiles_2022_unrestricted_SNAPSHOT_2023-10-24T16-39-44.zip`
```
Comment (hexylena) [inline topics/climate/tutorials/ocean-variables/tutorial.md:92]: where does this file come from? Is it in a zenodo somewhere or ?

**Comment by hexylena** [comment]
workflow tests should now pass as well :)

File: topics/climate/tutorials/ocean-variables/tutorial.md [hunk 2]
```diff
@@ -107,14 +102,13 @@ Once you finish your work on ODV, if you want to retrieve data and/or the entire
...
 > 
 >  ITS-90 water temperature and water body salinity variables have also been included ('as are') to complete the eutrophication and acidity data. If you use these variables for calculations, please refer to SeaDataNet for the quality flags: https://www.seadatanet.org/Products/Aggregated-datasets.
+> 
+> Regional datasets concerning eutrophication and acidity are automatically harvested, and the resulting collections are aggregated and quality-controlled using ODV Software and following a common methodology for all sea regions (https://doi.org/10.13120/8xm0-5m67). Parameter names are based on P35 vocabulary, which relates to EMODnet Chemistry aggregated parameter names and is available at: https://vocab.nerc.ac.uk/search_nvs/P35/.
```
[the remark this answers is not in the document]
Comment (Marie59) [reply topics/climate/tutorials/ocean-variables/tutorial.md:109]: will do !

File: topics/climate/tutorials/ocean-variables/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/climate/tutorials/ocean-variables/tutorial.md:92]: depending on the size, zenodo is also good! (We try and avoid having too-large files in the GTN where possible.)

Comment (Marie59) [reply topics/climate/tutorials/ocean-variables/tutorial.md:92]: It's big but I loaded it in the Fair-Ease galaxy

**Comment by Marie59** [comment]
I on myself have nothing else to add for this tuto. Do you think we can merge it or do it need some more review ? Are the data like that on github good ?

File: topics/climate/tutorials/ocean-variables/tutorial.md [hunk 3]
```diff
@@ -223,51 +228,114 @@ Now, if you have finished with your analysis you can exit ODV. To do so you need
...
+>
+> Click on **Run Tool**
+>
+> {% snippet faqs/galaxy/interactive_tools_open.md tool="ODV" %}
+>
+> 1. Click on close of the pop-up screen for the check for Updates
+> 2. Go the top left and click on **File**, then on **Open...**
+> 3. On the pop-up screen on the left panel select **ODV**, then the folder **galaxy**, then **data**. 
+> You should see a file open it (double clicking)
+> 4. Click on **Open** in the bottom right. Then click on **Next** every time and **Finish** you can keep everything by default.
```
Comment (bgruening) [inline topics/climate/tutorials/ocean-variables/tutorial.md:298]: @Marie59 I did not understand this sentence. What should I do with "Finish"?

File: topics/climate/tutorials/ocean-variables/tutorial.md [hunk 4]
```diff
@@ -179,7 +184,7 @@ Once you finish your work on ODV, if you want to retrieve data and/or the entire
 > You can now see bigger dots representing your data.
 > ![Image of your maps after the increase of the dots' size](../../images/ocean_var/big_dots.png)
 >
-> If you want to save it now that you already saved it once in the right folder outputs you just have to click lef on te save icon top left when it's red.
+> If you want to save it now that you already saved it once in the right folder outputs you just have to click left on the save icon top left when it's red.
```
Comment (bgruening) [inline topics/climate/tutorials/ocean-variables/tutorial.md:187]: I think this sentence is too complicated, at least for me. Can we simplify it a bit?

File: topics/climate/tutorials/ocean-variables/tutorial.md (hunk 3 above)
Comment (Marie59) [reply topics/climate/tutorials/ocean-variables/tutorial.md:298]: Click on it ;)

**Review by bgruening — APPROVED** [review]

**Pull request merged**